extras/rya.geoindexing/geo.common/src/main/java/org/apache/rya/indexing/GeoIndexerType.java

```diff
@@ -31,31 +28,49 @@
     /**
      * Geo Mesa based indexer.
      */
-    GEO_MESA(GeoMesaGeoIndexer.class),
+    GEO_MESA("org.apache.rya.indexing.accumulo.geo.GeoMesaGeoIndexer"),
```

Comment:
Would recommend a JIRA for adding the Service Loader (or other solution) be referenced to this.

Reply:
JIRA didn't exist.  Added RYA-354.